tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Models/Summary/OrderSummaryModelTests.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
+using System.Linq;
```

Comment:
We shouldn't need to add `System.Linq` and I think we can remove `System.Collections.Generic` too.